system_test/helpers/aest_docker.erl

```diff
@@ -450,6 +448,19 @@ attempt_epoch_stop(#{container_id := ID, hostname := Name} = NodeState, Timeout)
     end,
     ok.
 
+%% Sometime the stop command do not succeed...
+retry_epoch_stop(_NodeState, _ID, _Cmd, _Opts, 0) -> error;
```

Comment:
Where does the atom `error` go? Is it checked for anywhere? If this is the public API we should use an exception instead

Reply:
I understand I'll throw an error.